config/intellij-idea-inspections.xml

```diff
@@ -1926,7 +1926,7 @@ isolated classes and we cannot put them to separate package as it will affect us
     <inspection_tool class="SpringBeanAttributesInspection" enabled="true" level="ERROR" enabled_by_default="true" />
     <inspection_tool class="SpringBeanAutowiringInspection" enabled="true" level="WARNING" enabled_by_default="true" />
     <inspection_tool class="SpringBeanConstructorArgInspection" enabled="true" level="WARNING" enabled_by_default="true" />
-    <inspection_tool class="SpringBeanDepedencyCheckInspection" enabled="true" level="WARNING" enabled_by_default="true" />
+    <inspection_tool class="SpringBeanDependencyCheckInspection" enabled="true" level="WARNING" enabled_by_default="true" />
```

Comment:
This is coming from somewhere else, although, I can't figure out where the upstream is. Anyone know?

https://github.com/JetBrains/intellij-community/blob/4fd3db662e99eec9837c6178b77c6800d2dd6dbd/.idea/inspectionProfiles/idea_default_ci.xml#L3703

Reply:
please do not touch this, it is name of class in IntellijIdea, they will not update it :)